src/database.ts

```diff
@@ -1742,13 +1945,16 @@ class Database extends ServiceObject {
    * region_tag:spanner_create_storing_index
    * Creating a storing index:
    */
-  updateSchema(statements, callback) {
-    if (!is.object(statements)) {
+  updateSchema(
+    statements: Schema,
+    callback: UpdateSchemaCallback
+  ): Promise<[GaxOperation, databaseAdmin.longrunning.IOperation]> {
```

Comment:
We should export this as `UpdateSchemaResponse`

Reply:
Done!!